packages/core/src/AbstractRenderer.js

```diff
@@ -57,6 +55,14 @@ export default class AbstractRenderer extends EventEmitter
         // Add the default render options
         options = Object.assign({}, settings.RENDER_OPTIONS, options);
 
+        if (options.roundPixels)
+        {
+            // eslint-disable-next-line no-console
+            deprecation('5.0.0', 'roundPixels is no longer a valid configuration setting passed to the renderer. '
+            + 'Instead, each sprite/mesh has its own roundPixels property. '
+            + 'Change PIXI.settings.ROUND_PIXELS to set the default roundPixels value for newly created sprites/meshes.');
+        }
```

Comment:
We have not found a good way to deprecate things like options, which are inline. However, I like where you're going with this. The v4 deprecation function is here:

https://github.com/pixijs/pixi.js/blob/next/bundles/pixi.js/src/deprecated.js#L1-L48

I'm not a huge fan of having two functions that do the same thing. For the sake of getting this PR through, why don't you add a TODO inline comment to add a deprecation notice when detecting `options.roundPixels`. In another PR, we can generalize the deprecation function, because I think that's a little more complicated because of the `slice` and the differences in function signatures.